opentracing-api/src/main/java/io/opentracing/Tracer.java

```diff
@@ -113,6 +138,17 @@
          */
         SpanBuilder addReference(String referenceType, SpanContext referencedContext);
 
+        /**
+         * Remove any explicit (e.g., via {@link SpanBuilder#addReference(String,SpanContext)}) or implicit (e.g., via
+         * {@link ActiveSpanSource#activeContext()}) references to parent / predecessor SpanContexts, thus making the built
+         * Span a "root" of a Trace tree/graph.
```

Comment:
I'm fine with that. I like `asRoot()` to be more generally applicable as a flag that affects all references added to this span. Ignoring active span does feel like a separate concern.